docs/src/ci.md

```diff
@@ -293,15 +293,70 @@ Pipelines support [running containerized
 jobs](https://docs.microsoft.com/en-us/azure/devops/pipelines/process/container-phases?view=azure-devops).
 Alternatively, you can use [Command line tools](./cli.md#install-system-dependencies) to install all necessary dependencies.
 
+For running the Playwright tests use this pipeline task:
 ```yml
-pool:
-  vmImage: 'ubuntu-20.04'
-
-container: mcr.microsoft.com/playwright:v1.26.0-focal
-
-steps:
-...
+jobs:
+    - deployment: Run_E2E_Tests
+      pool:
+        vmImage: ubuntu-20.04
+      container: mcr.microsoft.com/playwright:v1.26.0-focal
+      environment: testing
+      strategy:
+        runOnce:
+          deploy:
+            steps:
+            - checkout: self
+            - task: Bash@3
+              displayName: 'Run Playwright tests'
+              inputs:
+                workingDirectory: 'my-e2e-tests'
+                targetType: 'inline'
+                failOnStderr: true
+                script: |
+                  export CI=true
+                  npm install
```

Comment:
```suggestion
                  npm ci
```

shouldn’t it be npm ci instead?

Reply:
Thanks. I've learned something today. I haven't been aware of `npm ci`.